packages/hooks/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './hooks';
+export * from './Provider';
```

Comment:
Similar to previous comment:

```suggestion
export { useWallet } from './hooks';
export {
  Provider,
  ProviderProps,
  Web3Context,
  Web3ContextType,
} from './Provider';
```